src/win32/sys_cod4linker_win32.c

```diff
@@ -44,6 +44,7 @@
 #include <inttypes.h>
 #include <direct.h>
 #include <windows.h>
+#include "../win32/win32_usleep.h"
```

Comment:
You can just import this as `#include "win32_usleep.h"`.

Reply:
Intresting :D do you also not know how to use code suggestions :P 
```suggestion
#include "win32_usleep.h"
```